Tests/SwiftFSMTests/FSMTests/SimplifiedPublicTests.swift

```diff
@@ -0,0 +1,53 @@
+import XCTest
+import SwiftFSM
+
+final class SimplifiedPublicTests: XCTestCase {
```

Comment:
Change file and test class name to PublicInterfaceTests, to accompany some changes to other file names that I made in preparation for this (PublicTests -> PublicFSMTests, FSMTests -> InternalFSMTests).

Reply:
(make sure to pull the latest changes from master first)